libcodechecker/analyze/analyzers/analyzer_clangsa.py

```diff
@@ -160,35 +157,35 @@ def construct_analyzer_cmd(self, result_handler):
             LOG.error(ex)
             return []
 
+    def get_xtu_dir(self):
+        config = self.config_handler
+        env = analyzer_env.get_check_env(config.path_env_extra,
+                                         config.ld_lib_path_extra)
+        triple_arch = ctu_triple_arch.get_triple_arch(self.buildaction,
+                                                      self.source_file,
+                                                      config, env)
+        xtu_dir = os.path.join(config.ctu_dir, triple_arch)
+        return xtu_dir
+
     def get_analyzer_mentioned_files(self, output):
         """
         Parse ClangSA's output to generate a list of files that were mentioned
```

Comment:
Technically, yes. It still parses the output, just with a different pattern.

Reply:
But it does not collect every mentioned file in the warnings but only files mentioned by a specific log message.

Reply:
We don't need anymore to catch the warnings. If there would be a warning from a TU, that must had been imported by Clang previously (together with the proper `CTU loaded` log).

Reply:
perhaps a better name could be: `get_analyzer_ctu_loaded_files` ?

Reply:
We can't just rename this function, because this is an inherited abstract function from `SourceAnalyzer`. And it is used in `analyzer_clang_tidy.py` as well.